pkg/toolset/toolset.go

```diff
@@ -251,3 +251,18 @@ func getGracefulStopContext() context.Context {
 
 	return ctx
 }
+
+func ProtocolParameters(networkName string, networkPrefix iotago.NetworkPrefix) *iotago.ProtocolParameters {
+	return &iotago.ProtocolParameters{
+		Version:     2,
+		NetworkName: networkName,
+		Bech32HRP:   networkPrefix,
+		MinPowScore: 0,
+		RentStructure: iotago.RentStructure{
+			VByteCost:    500,
```

Comment:
Maybe better set everything to zero, so we can also merge databases for networks with lower values.